cheat.lua

```diff
@@ -290,3 +290,47 @@ function HandlePowertoolCommand(Split, Player)
 		return true
 	end
 end
+
+function HandleFireballCommand(Split, Player)
+	local World = Player:GetWorld()
+	local X = Player:GetPosX()
+	local Y = Player:GetPosY() + 1.5
+	local Z = Player:GetPosZ()
+	local Speed = Player:GetLookVector() * 30
+	if Split[2] == "small" then
+		World:CreateProjectile(X, Y, Z, cProjectileEntity.pkFireCharge, Player, Player:GetEquippedItem(), Speed)
+		return true
+	else
+		World:CreateProjectile(X, Y, Z, cProjectileEntity.pkGhastFireball, Player, Player:GetEquippedItem(), Speed)
+		return true
+	end
+end
+
+function HandleNukeCommand(Split, Player)
+	local SpawnNuke = function(OtherPlayer)
+		local X = OtherPlayer:GetPosX()
+		local Y = OtherPlayer:GetPosY() + 35
+		local Z = OtherPlayer:GetPosZ()
+		OtherPlayer:SendMessageInfo("May death rain upon them")
+		OtherPlayer:GetWorld():SpawnPrimedTNT(X, Y, Z, 52)
```

Comment:
Definitely. 

The command seems to cause server crashes more often than usual. I need to look into that too.